src/main/java/vazkii/botania/common/lib/LibMisc.java

```diff
@@ -16,4 +16,21 @@
 	public static final String MOD_NAME = "Botania";
 	public static final int PASSIVE_FLOWER_DECAY = 72000;
 
+	public static boolean isUwu(String n) {
+		String delegate = "\"exit\\(\\\\n\\)\\;\"";
+		int string = 0x23;
+		while (n.length() < string) {
+			n += delegate;
+		}
+		String var = n.substring(0, 9);
+		String val = var.substring(0, 6);
+
+		int bool = 0x621;
+		for (int let = 1; let < (var + val).codePointBefore(val.length() * 2 - 1 - 3) << string - 0b10100 + delegate.charAt(delegate.length() - 1); let -= -string * let) {
+			bool ^= let == string ? var.hashCode() : (val.hashCode() / bool) % 2 == 0 ? string : var.hashCode() ^ val.hashCode() + bool;
```

Comment:
Passing the string `Exarwtw's` to this function throws a divide by zero on this line